.size-snapshot.json

```diff
@@ -1,21 +1,17 @@
 {
   "dist/react-powerplug.umd.js": {
-    "bundled": 20071,
-    "minified": 8068,
-    "gzipped": 2173
+    "bundled": 23877,
+    "minified": 9888,
+    "gzipped": 2560
   },
   "dist/react-powerplug.cjs.js": {
-    "bundled": 17186,
-    "minified": 8690,
-    "gzipped": 2055
+    "bundled": 20772,
+    "minified": 10750,
+    "gzipped": 2409
   },
   "dist/react-powerplug.esm.js": {
-    "bundled": 16589,
-    "minified": 8186,
-    "gzipped": 1936,
-    "treeshaked": {
-      "rollup": 1489,
-      "webpack": 1927
```

Comment:
Can you please remove size snapshot and run `yarn build` again?

Reply:
:heavy_check_mark:

Reply:
generated it
```
{
  "dist/react-powerplug.umd.js": {
    "bundled": 24185,
    "minified": 10002,
    "gzipped": 2571
  },
  "dist/react-powerplug.cjs.js": {
    "bundled": 21058,
    "minified": 10885,
    "gzipped": 2423
  },
  "dist/react-powerplug.esm.js": {
    "bundled": 20412,
    "minified": 10336,
    "gzipped": 2296
  }
}
```

Reply:
Probably this pr was open to early. Can you `git rebase master`?

Reply:
Ok! I'll do it

Reply:
I think now it's okay.